Userland/Libraries/LibCrypto/Hash/SHA1.cpp

```diff
@@ -67,6 +83,188 @@ inline void SHA1::transform(u8 const* data)
     secure_zero(blocks, 16 * sizeof(u32));
 }
 
+#if ARCH(I386) || ARCH(X86_64)
+sha1_attribute_target_x86 static void transform_impl(u32* state, u8 const* data)
+{
+    __m128i reverse_8;
+    __m128i abcd_1;
+    __m128i e;
+    __m128i old_abcd;
+    __m128i old_e;
+    __m128i msg_0;
+    __m128i abcd_2;
+    __m128i msg_1;
+    __m128i msg_2;
+    __m128i msg_3;
```

Comment:
Any reason not to put some of these variables in an array, or even better, an AK::Array?

Reply:
I would rather not. That would mean having two-element array named `abcd` and four element array `msg`. For the `abcd` that (according to me) does not make sense. The other is doable. If you wish, I will do that. If you insist I will do that `abcd` too.